src/sql/migrations.rs

```diff
@@ -596,6 +596,19 @@ CREATE INDEX smtp_messageid ON imap(rfc724_mid);
         )
         .await?;
     }
+    if dbversion < 92 {
+        sql.execute_migration(
+            r#"CREATE TABLE reactions (
+              msg_id INTEGER NOT NULL, -- id of the message reacted to
+              contact_id INTEGER NOT NULL, -- id of the contact reacting to the message
+              reaction TEXT DEFAULT '' NOT NULL, -- a sequence of emojis separated by spaces
+              PRIMARY KEY(msg_id, contact_id),
+              FOREIGN KEY(msg_id) REFERENCES msgs(id) ON DELETE CASCADE -- delete reactions when message is deleted
+              FOREIGN KEY(contact_id) REFERENCES contacts(id) ON DELETE CASCADE -- delete reactions when contact is deleted
+            )"#,
```

Comment:
wondering if we should add a timestamp as well.

i checked UI of other messengers, they do not show these informations, so it is probably not needed in UI.

but it can become important for checking which the most recent reaction is when reaction arrive in wrong order (cmp. https://github.com/deltachat/deltachat-core-rust/pull/2642) - or is that point already targeted differently?

Reply:
agree.